torchrl/envs/common.py

```diff
@@ -708,7 +712,7 @@ def policy(td):
                 exclude_action=False,
             )
             if not break_when_any_done and done.any():
-                _reset = done.view(tensordict.shape)
+                _reset = done
```

Comment:
I guess we'd like to clone it to be 100% sure that any downstream modification won't have surprising effects

Reply:
makes sense, i didn't add it as it was not there before but seems needed